src/test/java/org/jenkinsci/plugins/workflow/libs/LibraryMemoryTest.java

```diff
@@ -55,6 +60,7 @@ public class LibraryMemoryTest {
     @Rule public GitSampleRepoRule sampleRepo = new GitSampleRepoRule();
     @Rule public LoggerRule logging = new LoggerRule().record(CpsFlowExecution.class, Level.FINER);
 
+    // Using a set to avoid duplicate elements, which just makes `assertGC` slower.
     private static final List<WeakReference<ClassLoader>> LOADERS = new ArrayList<>();
```

Comment:
Not sure I understand this comment preceding a `List`.

Reply:
Accidentally left in from an old refactoring, while I was trying to diagnose why `loaderReleased` was passing, but doing so extremely slowly. In the end it started passing quickly without any clear reason and so I backed those changes out, but forgot to delete the comment.